common/zoom/zoom-scheduled-meeting.ts

```diff
@@ -0,0 +1,133 @@
+import { getAccessToken } from './zoom-authorization';
+
+type ZoomMeeting = {
+    next_page_token: string;
+    page_count: number;
+    page_number: number;
+    page_size: number;
+    total_records: number;
+    meetings: [
+        {
+            agenda: string;
+            created_at: string;
+            duration: number;
+            host_id: string;
+            id: number;
+            join_url: string;
+            pmi: string;
+            start_time: string;
+            timezone: string;
+            topic: string;
+            type: number;
+            uuid: string;
+        }
+    ];
+};
+
+const zoomUsersUrl = 'https://api.zoom.us/v2/users';
+const zoomMeetingUrl = 'https://api.zoom.us/v2/meetings';
+const zoomMeetingReportUrl = 'https://api.zoom.us/v2/report/meetings';
+const grantType = 'account_credentials';
+
+const createZoomMeeting = async (zoomUserId: string, startTime: Date, endDateTime?: Date) => {
+    try {
+        const { access_token } = await getAccessToken();
+
+        const response = await fetch(`${zoomUsersUrl}/${zoomUserId}/meetings`, {
+            method: 'POST',
+            headers: {
+                Authorization: `Bearer ${access_token}`,
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                agenda: 'My Meeting',
+                default_password: false,
+                duration: 60,
+                start_time: startTime,
+                timezone: 'Europe/Berlin',
+                type: 2,
```

Comment:
What is this type? A comment or enum would be nice

Reply:
You are right. A enum serves the purpose.